### PR TITLE
Testing the Reboot reset

### PR DESCRIPTION
**Pull request by chrisfinazzo**
Based on discussion in #7120.

### CODE REVIEW

File: docs/_sass/_reboot.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+/*# sourceMappingURL=bootstrap-reboot.min.css.map */
```
Comment (pathawks) [inline docs/_sass/_reboot.scss]: Are we actually generating a source map somewhere?

**Comment by DirtyF** [comment]
Could you provide a diff image of the change to see how this affects the current website?

Looks like Reboot does more things then Normalize https://codepen.io/ncerminara/embed/RLMwmy?height=500&slug-hash=RLMwmy&default-tabs=html,result&host=https://codepen.io&embed-version=2

File: docs/_sass/_reboot.scss (hunk 1 above)
Comment (chrisfinazzo) [reply docs/_sass/_reboot.scss]: Not that I'm aware of.

Does cURL permit something like `sourcemap=none` which would suppress this? I've used it when converting by hand, but I'm not sure it applies to a process like Rake tasks 🤔

**Comment by chrisfinazzo** [comment]
@DirtyF OK, will do.

File: docs/_sass/_reboot.scss (hunk 1 above)
Comment (pathawks) [reply docs/_sass/_reboot.scss]: Oh, I see: This is from the pre-generated CSS file that we are downloading.
No, not much we can do about this, I think.

Comment (chrisfinazzo) [reply docs/_sass/_reboot.scss]: @pathawks Yeah, given how this is structured, it's redundant.

**Comment by chrisfinazzo** [comment]
Sidebar margins and headers seem to be affected - looks like `margin-top` is 0 in a bunch of places and `h2`'s are sized as titles. We would need to override these, so it's not a drop-in replacement at this point.

**Reboot.css**
screen shot 2018-07-09 at 3 26 31 pm

**Normalize.css**
screen shot 2018-07-09 at 3 30 18 pm

**Comment by DirtyF** [comment]
@chrisfinazzo thanks for the feedback. Good to know, another thing to consider is the file size:

- Normalize 5.91 KB 
- Reboot  11.2 KB

Given that Reboot is bigger and needs additional rules, without bringing any gain to the website users the moment, I'll close this for now, we might reconsider our reset when we rewrite CSS, but it's far from our priorities at the moment 😄

**Pull request closed without merging**